mappings/net/minecraft/item/ItemStack.mapping

```diff
@@ -12,42 +12,61 @@ CLASS axx net/minecraft/item/ItemStack
 	FIELD k lastCheckedCanHarvestResult Z
 	FIELD l lastCheckedCanPlaceBlock Lbri;
 	FIELD m lastCheckedCanPlaceResult Z
+	METHOD <init> (Lbde;)V
+		ARG 1 provider
 	METHOD <init> (Lbde;I)V
-		ARG 1 container
+		ARG 1 provider
+		ARG 2 size
+	METHOD <init> (Lhx;)V
+		ARG 1 tag
 	METHOD A getRepairCost ()I
 	METHOD B toTextComponent ()Lji;
+		ARG 1 style
 	METHOD C getUpdateCooldown ()I
-	METHOD D getAmount ()I
+	METHOD D getSize ()I
```

Comment:
"size" implies a maximum.

Reply:
In general, collections in Java have a size and can be expanded and contracted as needed. It does not represent a fixed value.

Reply:
And even so, stacks do have a size maximum?